Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris.py

```diff
@@ -1,771 +1,3020 @@
+    if not activity_series.get('activityConnection', {}).get('nodes', []):
+        return CommandResults(readable_output=MESSAGES["NO_RECORDS_FOUND"].format("radar analysis status"))
+
+    context, hr = prepare_context_hr_radar_analysis_status(activity_series, activity_series_id, cluster_id)
+
+    return CommandResults(outputs_prefix=OUTPUT_PREFIX['RADAR_ANALYSIS_STATUS'],
+                          outputs_key_field="ActivitySeriesId",
+                          readable_output=hr,
+                          outputs=context,
+                          raw_response=raw_response)
+
+
+def sonar_sensitive_hits_command(client: PolarisClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    Collect sensitive hits object information.
+
+    :type client: PolarisClient
+    :param client: Rubrik Polaris client to use
+
+    :type args: ``dict``
+    :param args: arguments obtained from demisto.args()
+
+    :return: CommandResult object
+    """
+    incident = demisto.incident().get("CustomFields")
+
+    # objectName is an optional value for the command. When not set,
+    # look up the value in the incident custom fields
+    object_name = args.get('objectName', None)
+    if not object_name:
+        try:
+            object_name = incident.get("rubrikpolarisobjectname")
+        except AttributeError:
+            pass
+
+    search_time_period = arg_to_number(args.get('searchTimePeriod', 7))
+    response = client.get_sensitive_hits(search_time_period=search_time_period, object_name=object_name)
+    data = response.get("data", {}).get('policyObj')
+    if not data:
+        return CommandResults(readable_output=MESSAGES["NO_RECORDS_FOUND"].format("sensitive hits"))
+
+    context, hr = prepare_context_hr_sonar_sensitive_hits(data)
+    headers = ['ID', 'Total Hits']
+    readable_output = tableToMarkdown(name="Sensitive Hits", t=hr, headers=headers, removeNull=True)
+    return CommandResults(
+        outputs_prefix=OUTPUT_PREFIX['SONAR_SENSITIVE_HITS'],
+        outputs_key_field='id',
+        outputs=context,
+        raw_response=response,
+        readable_output=readable_output
+    )
+
+
+def rubrik_polaris_object_search_command(client: PolarisClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    Search for Rubrik discovered objects of any type, return zero or more matches.
+
+    :type client: ``PolarisClient``
+    :param client: Rubrik Polaris client to use
+
+    :type args: ``dict``
+    :param args: arguments obtained from demisto.args()
+
+    :return: CommandResult object
+    """
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    sort_by = args.get('sort_by', DEFAULT_SORT_BY)
+    sort_order = args.get('sort_order', DEFAULT_SORT_ORDER)
+    object_name = validate_required_arg("object_name", args.get('object_name'))
+
+    if not limit or limit <= 0 or limit > 1000:
+        raise ValueError(ERROR_MESSAGES['INVALID_LIMIT'].format(limit))
```

Comment:
maybe instead of raising an error we can set the limit to default value and just log it instead?

Reply:
```suggestion
    if not limit or limit <= 0 or limit > 1000:
        demisto.debug(f'limit is not valid. setting the limit to {DEFAULT_LIMIT}')
        limit = DEFAULT_LIMIT
```

Reply:
This change would lead to the user not being aware of the invalid value provided and he/she might assume that there are only 50 (default value of limit) entries. He/she might also think that invalid values are not handled and might contact customer support for bug resolution.